### PR TITLE
01_maven

### PR DESCRIPTION
**Pull request by misvaleris**
02_git

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea
```
Comment (artemmoroz) [inline .gitignore:1]: please take a look at this site  https://www.toptal.com/developers/gitignore and create more specific ignore file

File: parent/pom.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (artemmoroz) [inline parent/pom.xml]: you do not need a specific parent module (you already have one, ./pom.xml)

**Review by artemmoroz — CHANGES_REQUESTED** [review]

**Comment by artemmoroz** [comment]
also, please change groupId to some more relevant, e.g by.issoft.store or smthing similar

File: .gitignore (hunk 1 above)
Comment (misvaleris) [reply .gitignore:1]: @artemmoroz done, please review

File: parent/pom.xml (hunk 2 above)
Comment (misvaleris) [reply parent/pom.xml]: @artemmoroz done, please review

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>by.issoft.store</groupId>
```
Comment (artemmoroz) [inline pom.xml:7]: you should change groupId in all pom files)

**Review by artemmoroz — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (misvaleris) [reply pom.xml:7]: oh, my bad
done

**Review by artemmoroz — APPROVED** [review]

**Pull request merged**